src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+/**
+ * Encapsulates the User Interface.
+ * Handles the user interactions which are not relevant to the main program.
+ */
+
+public class Ui {
+
+    /**
+     * Greets the user when Duke is opened.
+     * @return A string to greet the user.
+     */
+    public String welcome() {
+        return "Hello! I'm duke. What can I do for you?";
+    }
+
+    /**
+     * Used to wrap around the output.
+     */
+    public void line() {
```

Comment:
It may be beneficial to include a verb instead of having one noun to name this method, since it completes an action. For instance, including the verb "print" in the name could improve its naming.